src/components/fieldset/fieldset.yaml

```diff
@@ -1,3 +1,38 @@
+params:
```

Comment:
do we need to think about `caller` for fieldset, or can this be done later

Reply:
I've added an explanation of `caller` for fieldset, have a look. It strikes me that we could probably do with an explanation of `caller` in our docs that we could then link to from the fieldset table.